checks/code_review_test.go

```diff
@@ -275,6 +282,247 @@ func TestCodereview(t *testing.T) {
 			},
 		},
 	}
+	probeTests := []struct {
```

Comment:
Could this be moved to a separate test function for clarity/maintainability?

Reply:
Done